### PR TITLE
Adding riverrem

### PR DESCRIPTION
**Pull request by klarrieu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by klarrieu** [comment]
@conda-forge-admin, please ping team.

I confirm I am willing to be the listed maintainer, and I believe this is now ready to merge.

**Comment by klarrieu** [comment]
@conda-forge-admin, please ping conda-forge/staged-recipes 

I confirm I am willing to be the listed maintainer, and I believe this is now ready to merge.

**Comment by klarrieu** [comment]
@conda-forge/help-python, ready for review, thanks!

**Comment by klarrieu** [comment]
@conda-forge-admin, please ping conda-forge/help-python

Ready for review, thanks!

**Comment by klarrieu** [comment]
@conda-forge-admin, please ping conda-forge/help-python

Ready for review, thanks!

File: recipes/riverrem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/riverrem/meta.yaml:1]: Ping! Also, please try grayskull to create your recipes. It will make a recipe that is almost "merge ready."

**Pull request merged**